src/pages/__snapshots__/Flavors.test.js.snap

```diff
@@ -182,9 +182,19 @@ exports[`Page <Flavors /> renders correctly when stash is not loaded 1`] = `
       <div
         className="text-left ml-2 mb-4 col"
       >
-        <i
-          className="fa fas-toggle-off"
-        />
+        <label
```

Comment:
Would you mind changing Flavors.js to add a line mocking the ToggleButton component? That will simplify this part of the snapshot and avoid it breaking if the component logic changes again.